frontend-react/src/components/shared/Table/Table.tsx

```diff
@@ -0,0 +1,225 @@
+                  }
+              })
+            : rowData;
+    }, [activeColumn, rowData, sortOrder]);
+    return (
+        <>
+            <thead>
+                <tr>
+                    {columnHeaders.map((header) => {
+                        return (
+                            <SortableTableHeader
+                                columnTitle={header}
+                                activeColumn={activeColumn}
+                                sortOrder={sortOrder}
+                                setActiveColumn={setActiveColumn}
+                                setSortOrder={setSortOrder}
+                            />
+                        );
+                    })}
+                </tr>
+            </thead>
+            <tbody>
+                {sortedData.map((row) => {
+                    return (
+                        <tr>
+                            {row.map((data) => {
+                                return (
+                                    <td
+                                        dangerouslySetInnerHTML={{
+                                            __html: sanitizeHtml(data.content),
+                                        }}
+                                    />
+                                );
+                            })}
+                        </tr>
+                    );
+                })}
+            </tbody>
+        </>
+    );
+};
+
+export const Table = ({
+    borderless,
+    compact,
+    emptyState,
+    fullWidth,
+    scrollable,
+    sortable,
+    stackedStyle,
+    striped,
+    rowData,
+}: TableProps) => {
+    const classes = classnames("usa-table", {
+        "usa-table--borderless": borderless,
+        "usa-table--compact": compact,
+        "width-full": fullWidth,
+        "usa-table--stacked": stackedStyle === "default",
+        "usa-table--stacked-header": stackedStyle === "headers",
+        "usa-table--striped": striped,
+    });
+
+    const columnHeaders = [
+        ...new Set(rowData.flat().map((data: RowData) => data.columnKey)),
+    ];
+
+    return (
+        <div
+            className={styles.reportstreamTable}
+            {...(scrollable && {
+                className: `usa-table-container--scrollable ${styles.reportstreamTable}`,
+                tabIndex: 0,
+            })}
+        >
+            <table className={classes} data-testid="table">
+                {sortable ? (
+                    <SortableTable
+                        rowData={rowData}
+                        columnHeaders={columnHeaders}
+                    />
+                ) : (
+                    <>
+                        <thead>
+                            <tr>
+                                {columnHeaders.map((header) => {
+                                    return <th>{header}</th>;
+                                })}
+                            </tr>
+                        </thead>
+                        <tbody>
```

Comment:
There's duplicate code between the SortableTable and non-SortableTable with the same markup.  Can we make a TableBody component that share that functionality?

Reply:
With the optional `isHTML` boolean + the highlighting of cells for sortable tables, there isn't much markup that's exactly the same and we'd then have to pass extra booleans for `isSortable` since the contents of both the `<th>` and `<td>` (ie the majority of content of a table) are different.